### ISSUE TITLE
Editor stops receiving input after opening scene with visible dialog

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#59088]
### Godot version

6721290

### System information

W10

### Issue description

Title.

### Steps to reproduce

1. Create FileDialog
2. Set `visible` property to true
3. Reload the scene
4. Editor is frozen
5. Bonus: If you have restore scenes on load active, project is now unusable until you either remove the scene or edit tscn to hide the dialog

**Comment by akien-mga** [comment]
Possibly a regression from https://github.com/godotengine/godot/pull/58282.

**Comment by KoBeWi** [comment]
Ah, then it's half-duplicate of #58229 probably. Now the editor freezes only after reload.

**Comment by keptsecret** [comment]
If I recall when I made that fix, restoring the scene with a visible FileDialog on a new load didn't do this. I'll take a look.

**Comment by akien-mga** [comment]
Is this still reproducible in latest 4.0 betas?

**Comment by KoBeWi** [comment]
No.
